code/datums/gamemode/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -552,3 +552,74 @@
 		message_admins("Rejected catbeast ruleset. Not enough threat somehow??")
 		return FALSE
 	return TRUE
+
+//////////////////////////////////////////////
+//                                          //
+//             PLAGUE MICE                  ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//                                          //
+//////////////////////////////////////////////
+
+/datum/dynamic_ruleset/midround/from_ghosts/faction_based/plague_mice
+	name = "Plague Mice Invasion"
+	role_category = /datum/role/plague_mouse
+	enemy_jobs = list("Chief Medical Officer", "Medical Doctor", "Virologist")
+	required_enemies = list(2,2,2,2,2,2,2,2,2,2)
+	required_candidates = 1
+	var/max_candidates = 5
+	weight = 5
+	cost = 25
+	requirements = list(90,70,50,40,30,20,10,10,10,10)
+	high_population_requirement = 40
+	flags = MINOR_RULESET
+	my_fac = /datum/faction/plague_mice
+	logo = "plague-logo"
+
+
+
+/datum/dynamic_ruleset/midround/from_ghosts/faction_based/plague_mice/review_applications()
```

Comment:
there has to be a better way to get your setupdisease call than to paste most of the review apps proc, let's add something oop like a general OnReview proc or something

Reply:
it's not just the setupdisease call, I also don't want to spawn a human but a mouse, and I want the objective to be forged before finish_setup() is called.

Reply:
see notes I made in discord re: generate_ruleset_body